utils.go

```diff
@@ -181,31 +184,11 @@ func transform(device *gcpiotpb.Device) *CbDevice {
 			LastAccessedGatewayId:   device.GatewayConfig.LastAccessedGatewayId,
 			LastAccessedGatewayTime: getTimeString(device.GatewayConfig.LastAccessedGatewayTime.AsTime()),
 		},
-		Credentials:        parsedCreds,
-		LastConfigAckTime:  getTimeString(device.LastConfigAckTime.AsTime()),
```

Comment:
Similar to above, removed any fields that will cause the new version of the webhook to return a 400